File: Sum-of-First-N-Numbers.py
# my

class Solution:
    def seriesSum(self, n : int) -> int:
        # code here
        count=0
        for i in range(n+1):
            count+=i
        return count

#second

class Solution:
    def seriesSum(self, n : int) -> int:
        # code here
        return int((n*(n+1))/2)
        
print(Solution().seriesSum(5))

File: Sum-of-First-N-Numbers.js
//executed but accuracy 66% 
class Solution {
    /**
    * @param number n

    * @returns number
    */
    seriesSum(n) {
        // code here
        let count=0
        for(let i=0;i<n+1;i++){
            count+=i
        }
        return count
    }
}

//second 

class Solution {
    /**
    * @param number n

    * @returns number
    */
    seriesSum(n) {
        // code here
        return (n*(n+1))/2
    }
}

console.log(new Solution().seriesSum(6))
